### PR TITLE
packaging: scripts to prepare artifacts and payload image

### PR DESCRIPTION
**Pull request by mythi**
Closes: #30

### CODE REVIEW

**Review by YangLiang3 — APPROVED** [review]
Thanks Mikko

File: tools/packaging/build/agent-enclave-bundle/config.json
Comment (zhiwei-intel-h) [inline tools/packaging/build/agent-enclave-bundle/config.json:52]: The value for hard and soft limits should extend to a bigger value, such as 65535. Because when pulling an image, such as Redis, it will extend the resource limit  8192.

Comment (mythi) [reply tools/packaging/build/agent-enclave-bundle/config.json]: fixed, thanks!

**Comment by dcmiddle** [comment]
I only see Intel reviews on this PR. In the future let's make sure there are diverse reviews before merging.

@jiazhang0 @haosanzi @confidential-containers/enclave-cc-maintainers Any issues with this PR?

**Pull request merged**
